en/docs/reference/connectors/creatio-connector/creatio-connector-reference.md

```diff
@@ -0,0 +1,623 @@
+        "Id": "12d21632-8b20-45d9-942f-20fcbbf2957c",
+        "Name": "API Test",
+        "OwnerId": "410006e1-ca4e-4502-a9ec-e54d922d2c00",
+        "CreatedOn": "2022-04-07T09:37:58.5679414Z",
+        "CreatedById": "410006e1-ca4e-4502-a9ec-e54d922d2c00",
+        "ModifiedOn": "2022-04-07T09:37:58.5679414Z",
+        "ModifiedById": "410006e1-ca4e-4502-a9ec-e54d922d2c00",
+        "ProcessListeners": 0,
+        "Dear": "",
+        "SalutationTypeId": "00000000-0000-0000-0000-000000000000",
+        "GenderId": "00000000-0000-0000-0000-000000000000",
+        "AccountId": "e6574af1-3e92-4099-958e-e798f52ee016",
+        "DecisionRoleId": "00000000-0000-0000-0000-000000000000",
+        "TypeId": "00000000-0000-0000-0000-000000000000",
+        "JobId": "00000000-0000-0000-0000-000000000000",
+        "JobTitle": "Marketing manager",
+        "DepartmentId": "00000000-0000-0000-0000-000000000000",
+        "BirthDate": "0001-01-01T00:00:00Z",
+        "Phone": "",
+        "MobilePhone": "+1 213 566 34 22",
+        "HomePhone": "",
+        "Skype": "",
+        "Email": "test@gmail",
+        "AddressTypeId": "00000000-0000-0000-0000-000000000000",
+        "Address": "",
+        "CityId": "00000000-0000-0000-0000-000000000000",
+        "RegionId": "00000000-0000-0000-0000-000000000000",
+        "Zip": "",
+        "CountryId": "00000000-0000-0000-0000-000000000000",
+        "DoNotUseEmail": false,
+        "DoNotUseCall": false,
+        "DoNotUseFax": false,
+        "DoNotUseSms": false,
+        "DoNotUseMail": false,
+        "Notes": "",
+        "Facebook": "",
+        "LinkedIn": "",
+        "Twitter": "",
+        "FacebookId": "",
+        "LinkedInId": "",
+        "TwitterId": "",
+        "ContactPhoto@odata.mediaEditLink": "Contact(12d21632-8b20-45d9-942f-20fcbbf2957c)/ContactPhoto",
+        "ContactPhoto@odata.mediaReadLink": "Contact(12d21632-8b20-45d9-942f-20fcbbf2957c)/ContactPhoto",
+        "ContactPhoto@odata.mediaContentType": "application/octet-stream",
+        "TwitterAFDAId": "00000000-0000-0000-0000-000000000000",
+        "FacebookAFDAId": "00000000-0000-0000-0000-000000000000",
+        "LinkedInAFDAId": "00000000-0000-0000-0000-000000000000",
+        "PhotoId": "00000000-0000-0000-0000-000000000000",
+        "GPSN": "",
+        "GPSE": "",
+        "Surname": "Test",
+        "GivenName": "API",
+        "MiddleName": "",
+        "Confirmed": true,
+        "IsNonActualEmail": false,
+        "Completeness": 30,
+        "LanguageId": "6ebc31fa-ee6c-48e9-81bf-8003ac03b019",
+        "Age": 19,
+        "IsEmailConfirmed": false
+    }
+
+
+    ```
+    
+??? note "Update Contact"
+    The contactUpdate operation is  used to update an existing contact details in the CreatioCRM. 
+
+    If the given ID does not exist in the CreatioCRM, this will create a  new contact with the given ID. The message body for the contact update will be taken from the payload of existing message context.
```

Comment:
```suggestion
    If the given ID does not exist in the CreatioCRM, this will create a new contact with the given ID. The message body for the contact update will be taken from the payload of existing message context.
```